R/tm_file_viewer.R

```diff
@@ -1,16 +1,17 @@
-#' File Viewer Teal Module
+#' `teal` module: File viewer
 #'
 #' The file viewer module provides a tool to view static files.
-#' Supported formats include text formats, \code{PDF}, \code{PNG}, \code{APNG},
-#' \code{JPEG}, \code{SVG}, \code{WEBP}, \code{GIF} and \code{BMP}.
+#' Supported formats include text formats, `PDF`, `PNG` `APNG`,
+#' `JPEG` `SVG`, `WEBP`, `GIF` and `BMP`.
 #'
 #' @inheritParams teal::module
 #' @inheritParams shared_params
-#' @param input_path optional, (`list`) of the input paths to either: specific files of accepted formats,
-#'   a directory or a URL. The paths can be specified as absolute paths or relative to the running
-#'   directory of the application. Will default to current working directory if not supplied.
+#' @param input_path (`list`) of the input paths, optional. Each element can be:
 #'
-#' @export
+#' @inherit shared_params return
+#'
+#' Paths can be specified as absolute paths or relative to the running directory of the application.
+#' Default to the current working directory if not supplied.
```

Comment:
This cannot be under `@inherit` tag!

```suggestion
#' Paths can be specified as absolute paths or relative to the running directory of the application.
#' Default to the current working directory if not supplied.
#'
#' @inherit shared_params return
```

Reply:
ah, thanks

Reply:
It was my bad, as I did the PR on this file... oopsie :laughing: